code/game/gamemodes/gang/gang.dm

```diff
@@ -247,6 +248,20 @@ var/list/gang_colors_pool = list("red","orange","yellow","green","blue","purple"
 		gang_bosses += G.bosses
 	return gang_bosses
 
+/datum/game_mode/proc/shuttle_check()
+	if(forced_shuttle)
+		return
+	var/normies = 0 
+	for(var/mob/living/M in GLOB.player_list)
+		if(isliving(M) && !M.stat==DEAD && (!M.mind in get_all_gangsters()))
```

Comment:
and also, the ```isliving(M)``` isn't necessary, as the fact you've typecasted the loop as ```for(var/mob/living/M in GLOB.player_list)``` means that you'll only get ```/mob/living```s or subtypes of ```/mob/living```.

Also ```M``` should be ```L``` as this is a ```/living``` not a ```/mob```, just consistency.